src/components/Table/Internal/Body/Scroller.tsx

```diff
@@ -194,7 +194,7 @@ export const Scroller = React.forwardRef(
             path: IconName.mdiChevronLeft,
           }}
           onClick={() => onClick('left')}
-          ariaLabel={scrollLeftAriaLabel}
+          ariaLabel={scrollLeftAriaLabelText}
```

Comment:
these maybe work, but its come to my attention recently that aria attributes in react shouldn't be camelcased
https://reactjs.org/docs/accessibility.html
>Note that all aria-* HTML attributes are fully supported in JSX. Whereas most DOM properties and attributes in React are camelCased, these attributes should be hyphen-cased (also known as kebab-case, lisp-case, etc) as they are in plain HTML:

Reply:
They work :) Might not in React 19 or 20, so we've some time for cleanup.

Reply:
Now that I'm looking at this again, these are props which then get passed into the actual aria-label implementation. we're good here i bet